### ISSUE TITLE
Duplication issue on view--"See all publishers" (elmcip.net/publisher)

### ISSUE THREAD

**Issue by AlvaroSeica** [elmcip/elmcip#416]
The view created of "/publisher" contains a bug of duplication when a publisher has 2 addresses.

Example: https://elmcip.net/publisher?page=2 
(Cf. Polity record at https://elmcip.net/publisher/polity)

<img width="977" alt="publishers_and_journals___elmcip" src="https://user-images.githubusercontent.com/6121438/53333226-9174ba80-38f5-11e9-8188-15f7b4bde948.png">

**Comment by steinmb** [comment]
Good catch. It is a side effect of us listing `City` and `Country` for the records. Not a bug. This is how databases work by default. We could try and group by publisher though I not sure how the table content should then look? Would this work?

title | City | Country | Updated
----- | --- | ------- | --------------
Polity | Cambridge, Malden | United Kingdom, United States | 01.05.2018

**Comment by AlvaroSeica** [comment]
@steinmb ok I understand. Perhaps we do not need City, just Country.
